static/app/components/replays/canvasReplayerPlugin.tsx

```diff
@@ -296,11 +294,7 @@ export function CanvasReplayerPlugin(events: eventWithTime[]): ReplayPlugin {
         return;
       }
       const [event, replayer] = queueItem;
-      try {
-        processEvent(event, {replayer});
-      } catch (err) {
-        Sentry.captureException(err);
-      }
```

Comment:
Since `processEvent` is async function and we don't await it, exceptions do not actually get caught.